module/platform/OpenCR/HardwareIO/data/config/nugus1/HardwareIO.yaml

```diff
@@ -94,3 +94,9 @@ servos:
     direction: 1
     offset: -9 * pi / 180
     simulated: false
+
+servo:
```

Comment:
If this configuration is the same in every single one, just put it in the default config file rather than the specific robot settings